src/linux/backend-v4l2.h

```diff
@@ -12,6 +12,7 @@
 #include <cstring>
 
 #include <algorithm>
+#include <array>
```

Comment:
What compliler/toolchain are you using that requires this include ?

Reply:
It wasn't required, as it is transitively included through something else, but it is used directly and will break if that transitive include ever goes away. So just a random best-practice fix, but unrelated to this change if you want to remove it.